Source/WebInspectorUI/UserInterface/Views/GeneralStyleDetailsSidebarPanel.js

```diff
@@ -128,8 +128,6 @@ WI.GeneralStyleDetailsSidebarPanel = class GeneralStyleDetailsSidebarPanel exten
 
     initialLayout()
     {
-        this._showPanel(this._panel);
```

Comment:
oh wait actually i think this might cause issues cause now `this._panel` will be in a different spot in the DOM (i.e. last instead of first)

i think we could/should just remove `_showPanel` entirely by inlining `this.contentView.addSubview(this._panel)` here and `this.contentView.element.classList.add("has-filter-bar");` right after `this._filterBar` is created

Reply:
Yes, that sounds good. I inline the relevant contents of `_showPanel()` into `initialLayout()`, preserving DOM order.

I dropped the toggling of `WI.GeneralStyleDetailsSidebarPanel.FilterInProgressClassName` because there's no way a created filter bar can have active filters.